demos/noise_suppression_demo/cpp/main.cpp

```diff
@@ -81,19 +63,18 @@ struct RiffWaveHeader {
     int data_length;       // Raw data length
 };
 
-
 const unsigned int fourcc(const char c[4]) {
     return (c[3] << 24) | (c[2] << 16) | (c[1] << 8) | (c[0]);
 }
 
-void read_wav(const std::string& file_name, RiffWaveHeader& wave_header, std::vector<int16_t>& wave) {
-    std::ifstream inp_wave(file_name, std::ios::in|std::ios::binary);
+void read_wav(const string& file_name, RiffWaveHeader& wave_header, vector<int16_t>& wave) {
```

Comment:
I don't remember someone declared there was an agreement. We just abandoned this discussion. Thus I follow my point of view. Use namespaces explicitly except for std because it is extremely common.

Reply:
There's someone else using namespace: https://github.com/openvinotoolkit/open_model_zoo/pull/2866/files#diff-91dbcb865ed305331058ac9274bee1ee883e13a357b2763489215f73e27c8d8cR13 xD

(in header!!!)

Reply:
I personally have no objections against using namespaces, but agreement is an agreement. @vladimir-dudnik, do we reject using "using namespace" at all or just for OpenVINO classes ?

Reply:
I'm ok with both using or not using namespace. But I'm against hard requirement there. We need to have some freedom for authors to express their idea. Sometime we would like to highlight the fact the object comes from certain namespace (for example when using cv::String and std::string in one file it is better to make the difference more explicit). For OV2.0 API it is also better to provide explicit namespace to show (not to hide) the new things. Using `std::` was in original source and I do not see strong need to change it. What this gives us? What technical issue we solve with this change?

Reply:
It shortens lines in code

Reply:
yes it shortens. But there is always a balance between lines of code and code readability. I'm personally will always vote for better code readability (whatever that means)

Reply:
Reverted